lib/methods/veres-one/veres-did-doc.js

```diff
@@ -206,6 +206,106 @@ class VeresOneDidDoc {
     };
   }
 
+  /**
+   * Composes and returns a service id for a service name.
+   *
+   * @param serviceName {string}
+   *
+   * @returns {string} Service id
+   */
+  serviceIdFor(serviceName) {
+    if(!serviceName) {
+      throw new Error('Invalid service name');
+    }
+
+    return this.id + ';' + encodeURIComponent(serviceName);
```

Comment:
This is ok for now, but please file an issue about how we may be quickly revisiting it to be something like:

```js
return this.id + ';service=' + encodeURIComponent(serviceName);`
```

Reply:
Well, I mean, I can just change it to that right now :) s' not a very big change.